pkg/config/app_configuration.go

```diff
@@ -13,6 +13,13 @@ type ApplicationConfig struct {
 	// Duration. example: "30s"
 	ActorScanInterval string `json:"actorScanInterval"`
 	// Duration. example: "30s"
-	DrainOngoingCallTimeout string `json:"drainOngoingCallTimeout"`
-	DrainRebalancedActors   bool   `json:"drainRebalancedActors"`
+	DrainOngoingCallTimeout string           `json:"drainOngoingCallTimeout"`
+	DrainRebalancedActors   bool             `json:"drainRebalancedActors"`
+	Reentrancy              ReentrancyConfig `json:"reentrancy,omitempty"`
+}
+
+type ReentrancyConfig struct {
+	Enabled           bool     `json:"enabled"`
+	Methods           []string `json:"methods,omitempty"`
+	MaximumStackDepth *int     `json:"maximumStackDepth,omitempty"`
```

Comment:
This name is so much better. The only thing is to check if we use "Max" or "Maximum" in other places - just for consistency.

Reply:
We use 'max' not 'maximum', changed it to match.